app/templates/administrative-units/administrative-unit/core-data/index.hbs

```diff
@@ -25,240 +25,131 @@
-                      <p>{{@model.administrativeUnit.denomination}}</p>
-                    </div>
-                  </div>
-                </div>
-              {{/if}}
-            </div>
-            <div class="au-o-grid">
-              <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                <div class="au-o-grid">
-                  <div class="au-o-grid__item au-u-1-4">
-                    <p><strong>Grensoverschrijdend</strong></p>
-                  </div>
-                  <div class="au-o-grid__item au-u-3-4">
-                    <p>{{@model.administrativeUnit.crossBorderNominal}}</p>
-                  </div>
-                </div>
-              </div>
-            </div>
-            {{#if @model.administrativeUnit.scope.label}}
-              <div class="au-o-grid">
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>Werkingsgebied</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>{{@model.administrativeUnit.scope.label}}</p>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            {{/if}}
-            <div class="au-o-grid">
-              {{#each @model.administrativeUnit.identifiers as |identifier|}}
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>{{identifier.idName}}</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>{{identifier.structuredIdentifier.localId}}</p>
-                    </div>
-                  </div>
-                </div>
-              {{/each}}
-            </div>
-          </div>
-        </c.content>
-      </AuCard>
-    </div>
+    <DataCard>
+      <:title>{{@model.administrativeUnit.classification.label}}</:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Naam eredienst</:label>
+            <:content>{{@model.administrativeUnit.name}}</:content>
+          </Item>
+          <Item>
+            <:label>Status</:label>
+            <:content><OrganizationStatus @status={{@model.administrativeUnit.organizationStatus}} /></:content>
+          </Item>
+          <Item>
+            <:label>Type eredienst</:label>
+            <:content>{{@model.administrativeUnit.recognizedWorshipType.label}}</:content>
+          </Item>
+          {{#if @model.administrativeUnit.denomination}}
+            <Item>
+              <:label>Strekking</:label>
+              <:content>{{@model.administrativeUnit.denomination}}</:content>
+            </Item>
+          {{/if}}
+        </Grid>
+
```

Comment:
Cross border only exists for worship service, it's better if we include the conditional here?

Reply:
Ah, yea I can add that as well :+1: